test/verify/check-client

```diff
@@ -39,7 +39,7 @@ class TestClient(testlib.MachineCase):
         if self.m_target.image.startswith("debian") or self.m_target.image.startswith("ubuntu"):
             self.m_target.execute("dpkg --purge cockpit-pcp-dbgsym || true; dpkg --purge cockpit-pcp pcp")
         elif self.m_target.image != 'arch':
-            self.m_target.execute("rpm --erase --verbose cockpit-pcp pcp")
+            self.m_target.execute("rpm --erase --verbose cockpit-pcp pcp; systemctl daemon-reload")
```

Comment:
Is the "systemd not properly updated thing" an intended behaviour change or a bug?  If it's a bug (which I'd guess it is) then I'd prefer that we enable this gated by an OS image check and a `# HACK` comment...